packages/cli/src/util/config.ts

```diff
@@ -116,3 +128,13 @@ export async function loadFoundryConfig(): Promise<
 
   return undefined;
 }
+
+function parseConfigFile(
```

Comment:
Per the other comment I think it would be much better to keep these in sync by having a JSON parser, etc. and either a switch on the extension to call the parser or a map from extension to parser